### PR TITLE
Fixing deposit policy error

### LINKED ISSUE #50: Fixing Deposit policy error

### PR DESCRIPTION
**Pull request by cdsandoval**
Close #50 

Add search and create method to deposit policy.

### CODE REVIEW

File: db/migrate/20190503030315_add_country_to_senders.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddCountryToSenders < ActiveRecord::Migration[5.2]
+  def change
+    add_column :senders, :country, :string
+  end
+end
```
Comment (Linzeur) [inline db/migrate/20190503030315_add_country_to_senders.rb]: This migration was added in your previous PR, please you update your master and apply a rebase with your branch

File: db/migrate/20190503030335_add_city_to_senders.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddCityToSenders < ActiveRecord::Migration[5.2]
+  def change
+    add_column :senders, :city, :string
+  end
+end
```
Comment (Linzeur) [inline db/migrate/20190503030335_add_city_to_senders.rb]: The same that my before comment

File: db/seeds.rb [hunk 3]
```diff
@@ -89,7 +91,9 @@
 sendermailer = Sender.create(
   store_name: "Tienda prueba mailer",
   email: "diegotc86@gmail.com",
-  order_id: 1000
+  order_id: 1000,
+  city: "Lima",        
+  country: "Peru",
 )
 
 usermailer = User.create(
```
Comment (Linzeur) [inline db/seeds.rb]: The same that my before comment:
https://github.com/codeableorg/rails-fedex-clon/pull/51/files#r280660221

File: db/migrate/20190503030315_add_country_to_senders.rb (hunk 1 above)
Comment (cdsandoval) [reply db/migrate/20190503030315_add_country_to_senders.rb]: Already fix it !!

File: db/migrate/20190503030335_add_city_to_senders.rb (hunk 2 above)
Comment (cdsandoval) [reply db/migrate/20190503030335_add_city_to_senders.rb]: Yup, fixed it.

File: db/seeds.rb (hunk 3 above)
Comment (cdsandoval) [reply db/seeds.rb]: Changes are done buddy !!

**Pull request merged**